dex/src/main/scala/com/wavesplatform/dex/api/ws/protocol/WsAddressSubscribe.scala

```diff
@@ -72,7 +92,8 @@ object WsAddressSubscribe {
     clientId: String,
     firstTokenExpirationInSeconds: Long,
     activeTokenExpirationInSeconds: Long,
-    scope: List[String]
+    scope: List[String],
+    isDebug: Boolean = false
```

Comment:
`JwtPayload` is what we receive from an external service and it doesn't contain `isDebug`. Ideally we want to receive such information, but now it doesn't exist. So we must change `validate` to:
```scala
def validate(jwtPublicKey: String, networkByte: Byte): Either[MatcherError, (JwtPayload, Boolean)] =
```
where `true` means debug.

Reply:
fixed